guava/pom.xml

```diff
@@ -38,12 +38,13 @@ Guava (https://github.com/google/guava) types (currently mostly just collection
     <!-- 28-Feb-2023, tatu: 2.15 raises baseline from 21.x to 23.x -->
     <!-- 01-May-2023, tatu: 2.16 raises baseline from 23.x to 25.x -->
     <version.guava>25.1-jre</version.guava>
+    <version.guava.osgi-base>25.1</version.guava.osgi-base>
```

Comment:
As mentioned in https://github.com/FasterXML/jackson-datatypes-collections/issues/160#issuecomment-2340113552 the exact version that is used for exporting the required package in guava library is `25.1.0`. Thus using this exact version string we should be safe.